.circleci/config.yml

```diff
@@ -2,6 +2,29 @@ version: 2
 orbs:
   coveralls: coveralls/coveralls@1.0.4
 jobs:
+  lint:
```

Comment:
Instead of creating a separate job for linting, can you please add it as part of the build job? - run: npm run lint before npm run build.